app/decorators/drawing_decorator.rb

```diff
@@ -16,7 +16,7 @@ def story_concat
 
   private
 
-  def text_concat(text, max_length=30)
-    text[0, max_length - 1].concat('...')
+  def text_concat(text, max_length=100)
```

Comment:
The max length in @rocioleon's wireframes was quite short. Can we reduce this to 60?